### PR TITLE
Install "jq" package

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description
This PR adds installation of the `jq` package to this role.

## 💭 Motivation and Context
We want to use `jq` downstream as part of the process to automate the creation of Nessus users at deploy time.

## 🧪 Testing
All automated tests pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by mcdonnnj — APPROVED** [review]
Makes sense 👍

**Review by jsf9k — CHANGES_REQUESTED** [review]
No need for loop. Also, why not create a single variable for both packages?

File: tasks/main.yml [hunk 1]
```diff
@@ -64,8 +64,11 @@
...
 
-# "expect" is used downstream to automate creation of Nessus users
-- name: Install expect package
+# "expect" and "jq" are used downstream to automate creation of Nessus users
+- name: Install expect and jq packages
   package:
-    name: "{{ expect_package_name }}"
+    name: "{{ item }}"
     state: present
+  loop:
```
Comment (jsf9k) [inline tasks/main.yml]: No need for a loop here. package can take a list for name. Also, why not create a single variable for both packages?

Comment (dav3r) [reply tasks/main.yml]: Right- forgot that we can do that.  See https://github.com/cisagov/ansible-role-nessus/pull/11/commits/2cd00b99eb6c8052f957757186fd87f89785ca27 - thanks!

**Review by jsf9k — APPROVED** [review]
Looks good!

**Pull request merged**